kodi_addon_checker/check_addon.py

```diff
@@ -124,3 +127,15 @@ def _get_addons(xml_url: str):
         LOGGER.error(errrt)
     except requests.exceptions.ConnectTimeout as errct:
         LOGGER.error(errct)
+
+
+def check_schema(report: Report, Addon_xml, schema_path):
+    for xsd in os.listdir(schema_path):
```

Comment:
why?

Reply:
First I was thinking that we'll test a single addon.xml against all the schema that is why I added this loop.
But now I realized that we don't need it instead we will have to test every addon.xml according to their extension point right ?

Reply:
Can't you write one xsd schema which is correct for all addons?

Reply:
for all addons as in for all the extension point i.e `plugins`, `scripts`, `resources` ?

Reply:
Yes on schema which contains everything for plugins, script, resources, skins (and maybe more, don't know if we have more).

Reply:
okay I'll try to do so